### ISSUE TITLE
FIrst order gradients are used for outer loop optimization

### ISSUE THREAD

**Issue by sebamenabar** [uvm-neurobotics-lab/ANML#3]
Hi, there was a bug on the original code of OML where they did not create the computation graph for the inner loop updates. Looking the ANML code it seems to have the same issue, specifically [here](https://github.com/uvm-neurobotics-lab/ANML/blob/cabaaf7f2336496cd51065847c524a705408d562/model/meta_learner.py#L164) it is currently
```python3
grad = torch.autograd.grad(loss, fast_weights, allow_unused=False)
```
but to correctly backpropagate through the inner optimization it should be
```python3
grad = torch.autograd.grad(loss, fast_weights, allow_unused=False, create_graph=True)
```

I was wondering which version was used for the results on the paper, OML's author said fixing this bug improved performance and reduced training time.
Thanks

p.d.: congratz on the work, is really cool
